src/service/url-replacements-impl.js

```diff
@@ -812,6 +813,30 @@ export class UrlReplacements {
   }
 
   /**
+   * Append URL parameters from data-amp-addparams to URL
+   * @param {string} url
+   * @param {!Element} element An anchor element.
+   * @return {string} url
+   * @private
+   */
+  extendURLParams_(url, element) {
+    const additionalURLParamaters = element.getAttribute('data-amp-addparams');
+    if (!additionalURLParamaters) {
+      return url;
+    }
+    additionalURLParamaters.trim().split(/\s+/).forEach(urlParameter => {
```

Comment:
Added `spaceDelimitedParamsToMap` to `url.js` and used `addParamsToUrl`. Now because of the `url.js` usage it got simplified